library/ziti_ctrl.c

```diff
@@ -430,6 +430,9 @@ static void ctrl_body_cb(tlsuv_http_req_t *req, char *b, ssize_t len) {
     if (len > 0) {
         if (resp->resp_content == ctrl_content_json) {
             if (resp->content == NULL) {
+#ifdef ZITI_DEBUG
+                CTRL_LOG(DEBUG, "HTTP RESPONSE: %s", b);
```

Comment:
should this specific log message require an opt-in of some kind (e.g. env variable)?

Reply:
i thought that making someone compile it specifically with this option enabled was "opt-in enough"?

Reply:
in a sense that was my suggestion -- you opt-in by setting the log level VERBOSE or TRACE are more appropriate

Reply:
removed the #define, changed to debug, and used `%.*s`